packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -99,7 +100,8 @@ const INITIAL_STATE: State = {
   isError: false,
   styles: [],
   sections: [],
-  itemMap: { byItem: {}, bySection: {} }
+  itemMap: { byItem: {}, bySection: {} },
+  metadata: []
```

Comment:
I think we need to do similar for `attributes` and `relations` since we add that as part of the response json. Those two properties are required in the Invoices/POs